Makefile

```diff
@@ -33,7 +33,7 @@ else
 DOCKER_BUILD_ARGS += --build-arg VERSION="$(OPERATOR_TAG)"
 endif
 DOCKER_BUILD_ARGS += --build-arg REVISION="$(shell git rev-parse HEAD)"
-DOCKER_BUILD_ARGS += --build-arg BUILD_DATE="$(shell date --rfc-3339=seconds)"
+DOCKER_BUILD_ARGS += --build-arg BUILD_DATE="$(shell date -Iseconds)"
```

Comment:
Let's move this to a separate PR.

Reply:
Ok, moved here: https://github.com/ytsaurus/yt-k8s-operator/pull/260.